### PR TITLE
WIP add windows support

### PR DESCRIPTION
**Pull request by jbweston**

### CODE REVIEW

File: recipe/build_linux.conf [hunk 1]
```diff
@@ -1,7 +1,3 @@
-[lapack]
-library_dirs = PREFIX/lib
-libraries = openblas gfortran
-extra_link_args = -Wl,-rpath=PREFIX/lib
 [mumps]
 include_dirs = PREFIX/include
 library_dirs = PREFIX/lib
```
Comment (basnijholt) [inline recipe/build_linux.conf]: @jbweston you'll need to add `openblas` to the `[mumps]` section I think.

**Comment by basnijholt** [comment]
@conda-forge-admin, please rerender

**Comment by jbweston** [comment]
@basnijholt that rerender removed all the windows stuff!

**Comment by jbweston** [comment]
It claims the windows build succeeded, but if you look at the Appveyor logs you'll see that it didn't actually do anything!

**Comment by jbweston** [comment]
@conda-forge-admin, please rerender

**Comment by jbweston** [comment]
Ok it builds but there are 3 tests failing. The logs are all mangled (looks like appveyor might be running several things in parallel and interleaving the output) so it's difficult to tell what's going on, but it seems that what is failing is:

+ `test_rcond_from_lu`: there is a comment above the failing test that seems to imply that the test should not be too strict. I do not know enough about this specific issue to tell whether we should just disable the test
+ `test_symm_algorithm_equivalence`: Seems our check triggers some internal assertions withing numpy? A serious WTF going on in here.
+ `test_finalization`: the check that all the sites in the system are also in the builder fails. WTF

**Comment by jbweston** [comment]
To add insult to injury, appveyor for some reason decides that the build succeeds despite the failing tests: https://ci.appveyor.com/project/conda-forge/kwant-feedstock/build/1.0.29

**Comment by jbweston** [comment]
## Action plan

`test_rcond_from_lu`: we just have to decide whether this test is important. My vote would be just to disable it on windows

`test_symm_algorithm_equivalence`: the apparent exchange of the degrees of freedom could be due to assumptions we make about the way eigenvalues are ordered, maybe in some very degenerate case? We would need to look at the relevant code in Kwant to see where the problem could be, and investigate in isolation on our own machines (because modifying Kwant and getting conda forge to rebuild is far too slow).

`test_finalization`: could be due to the fact that we are comparing tinyarrays with tuples. On linux `assert tinyarray.array([1, 2]) in  {(1, 2)}` works, but maybe (for some crazy reason) this is not the case on windows

**Comment by jbweston** [comment]
@basnijholt you have a windows machine: we should block a few hours to debug this.

**Comment by basnijholt** [comment]
@conda-forge-admin, please rerender

**Comment by basnijholt** [comment]
@conda-forge-admin, please rerender

**Comment by basnijholt** [comment]
@conda-forge-admin, please rerender

**Comment by jbweston** [comment]
@basnijholt merge time :tada: :tada:

**Pull request merged**